### DISCUSSION TITLE
GitHub Sync not working

### DISCUSSION THREAD

**Discussion by arcuslinus** [GitbookIO/community#494] [0. General bugs and questions] [answered]
Hi there, 

unfortunately enabling GitHub sync doesn't work for me. When I try to select the GitHub installation using the arrow button, I get a error message "Something went wrong.".

<img width="552" alt="Screenshot 2024-02-19 at 20 59 35" src="https://github.com/GitbookIO/community/assets/42419545/c4a7a2e1-b56b-4626-a697-f0f61a8e090f">
<img width="496" alt="Screenshot 2024-02-19 at 20 59 39" src="https://github.com/GitbookIO/community/assets/42419545/2991b92f-bfd4-4d95-9ede-eaefd71b2737">

Any ideas how I can fix that?

**Comment by addisonschultz** [comment]
@linuskoester can you try fully uninstalling the integration and trying it again from scratch? 

Can you try this also with a brand new space?

**Reply by arcuslinus** [reply]
Hi @addisonschultz, 
thanks for the quick reply.

I already did both of that, serveral times. I also completely removed the extension from my GitHub account. I also tried it in diffrent browsers (Safari & Chrome). I even created a brand new GitBook account to test it, but the issue persisted :/

**Reply by addisonschultz** [reply]
Hey @linuskoester can you email me to addison @ gitbook.com with more info on where this is happening?

If you could send 
- Space ID
- Your user email 

I'll look into it right away!

**Reply by arcuslinus** [reply]
@addisonschultz Thank you so much, I wrote you an email

**Comment by addisonschultz** [comment, marked as answer]
This issue is now fixed! ❤️
